src/gridcoin/researcher.cpp

```diff
@@ -1141,15 +1109,19 @@ void Researcher::RunRenewBeaconJob()
 
 std::string Researcher::Email()
 {
-    std::string email = GetArgument("email", "");
+    std::string email = gArgs.GetArg("-email", "");
+
+    // If the investor mode flag is set, it should override the email setting. This is especially important now
+    // that the read-write settings file is populated, which overrides the settings in the config file.
+    if (gArgs.GetBoolArg("-investor", false)) email = std::string();
     boost::to_lower(email);
```

Comment:
```suggestion
    std::string email;
    
    // If the investor mode flag is set, it should override the email setting. This is especially important now
    // that the read-write settings file is populated, which overrides the settings in the config file.
    if (gArgs.GetBoolArg("-investor", false)) return email;
    
    email = gArgs.GetArg("-email", "");
    boost::to_lower(email);
```

Minor: can skip some unnecessary evaluation if `-investor`.